:wrench: issue #42 fix

therealsylaucoin opened the pull request:
**What was changed:**
- Removed #preview-text html code along with associated styles.
- Added #canvas styling to achieve the checkered background commonly associated with transparent.
- Added a .dark version of the checkered background as well.

**Things to know:**
- The colours of the checkered grade must remain fairly close to each other in terms of shade. When given too much of a contrast, the checkered grade takes on a warped look and is difficult on the eyes.
- I am a new FED and this is my first attempt at contributing to open source code, please provide any feedback pertaining to this pr (apologies for not including the commit emoji 🤦🏻).

therealsylaucoin committed:
issue #42 fix

mmaismma reviewed (changes requested):
You did an amazing job. I salute your logic and details! I did not mention dark mode but you implemented that anyway, fascinating!  
But since this is your first time contributing to open source, you missed some things.


As mentioned in our contributing guidelines, you had to make your changes in `Source/cubfan135/`. Please move all changes from `cubfan135/` to `Source/cubfan135`.

mmaismma commented on cubfan135/style.css:
```diff
@@ -197,6 +181,11 @@ body.dark #preview-area {
 }
 
+body.dark #canvas {
+  box-shadow: 0 0 14.4px 0 #151515, 0 2.2px 3.6px 0 #191919;
```
With this shadow:  
![image](https://user-images.githubusercontent.com/44745238/103471642-a4395f80-4da8-11eb-871a-61d5d32fe533.png)

Without this shadow (derives its shadow from under the `#canvas {`:  
![image](https://user-images.githubusercontent.com/44745238/103471645-b0252180-4da8-11eb-84dc-f493b6a638c7.png)

I think most will prefer the latter.

therealsylaucoin replied:
@mmaismma Now that I look at it, absolutely agree with the box-shadow. It's been removed. 
Thank you for your feedback, the requested changes have been made :)

therealsylaucoin committed:
:wrench: issue #42 fix - Requested changes implemented (moved to Source/cubfan135 + box-shadow tweek)

therealsylaucoin commented:
**What was changed:**

- Restored #preview-text html code along with associated styles in cubfan135
- Removed #preview-text html code along with associated styles from Source/cubfan135 (index and style)
- Added #canvas styling to achieve the checkered background commonly associated with transparent in Source/cubfan135.style.css
- Added a .dark version of the checkered background as well in Source/cubfan135.style.css
- Removed previously implemented box-shadow for .dark canvas

mmaismma reviewed (approved):
This masterpiece PR is merging!
